als-node-client/node-package/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aml-org/als-node-client",
-  "version": "5.1.0-SNAPSHOT",
+  "version": "5.1.0.1-SNAPSHOT",
```

Comment:
weird

Reply:
It was a test, shouldn't have been pushed